docs/discover/log-pattern-analysis.asciidoc

```diff
@@ -16,23 +14,19 @@ can use your own data.
 . If you don't see any results, expand the time range, for example, to
 *Last 15 days*.
 
-. Click the `message` field in the **Available fields** list sidebar and click
-**Run pattern analysis**.
-+
---
-[role="screenshot"]
-image::images/log-pattern-analysis-available-fields.png["Available fields view in Discover showing the message field selected."]
-
-The pattern analysis starts. The results are displayed in a flyout when the
-analysis is complete.
+. Click the **Patterns** tab next to **Documents** and **Field statistics**. The
+pattern analysis starts. The results are displayed under the chart. You can
+change the analyzed field by using the field selector.
```

Comment:
It might be worth mentioning the `Minimum time range` option in the settings menu.
This lets you widen the time range when calculating patterns, which improves pattern accuracy. But we still only show the patterns in the selected time range.

Reply:
Updated via [8a74ac1](https://github.com/elastic/kibana/pull/189876/commits/8a74ac1bb3c58cdb0fb250400d18a02ac1cb2205).